rpc/server.go

```diff
@@ -50,46 +58,70 @@ type Config struct {
 	XMRMaker        XMRMaker
 	ProtocolBackend ProtocolBackend
 	RecoveryDB      RecoveryDB
+	Namespaces      map[string]struct{}
```

Comment:
I think this would be more obvious if it was:
```go
EnabledNamespaces map[string]bool
```

Reply:
I think that's more complicated b/c the value can be true, false, or not set

Reply:
It is less complicated because you no longer have to check if the value was set. You assign a single value from the map instead of two: https://go.dev/play/p/z7_qyFbzkJ0

Reply:
hmm, i suppose. i always thought this way was more idiomatic for if something is either set or not set

Reply:
You are thinking that because, at least with earlier versions of go (and maybe even now), if you are managing enormous sets, you can get a very tiny performance increase using `struct{}`. In our use case, it is just a small map that isn't even accessed much. The performance win wouldn't even be measurable.

Reply:
The comment wasn't to change the type in isolation.  It was that the code would be more self-documenting if the map name was changed to `EnabledNamespaces`.  Then code that does a lookup in the map is asking a question: "Is this namespace enabled?" The answer to that question is a boolean, which lead to the type change suggestion. The looping code you are referring to can have its inner switch removed and just check if the namespaces it cares about are enabled.